integration/data/test_snapshot.py

```diff
@@ -366,3 +371,96 @@ def test_expansion_with_backing_file(grpc_backing_controller,  # NOQA
                                          VOLUME_BACKING_NAME,
                                          ENGINE_BACKING_NAME,
                                          read_from_backing_file(0, SIZE))
+
+
+def test_snapshot_mounted_filesystem(grpc_controller,  # NOQA
+                         grpc_replica1, grpc_replica2):  # NOQA
+    address = grpc_controller.address
+    dev = get_dev(grpc_replica1, grpc_replica2, grpc_controller)
+    snapshot_mounted_filesystem_test(VOLUME_NAME, dev, address, ENGINE_NAME)
+
+
+def snapshot_mounted_filesystem_test(volume_name, dev, address, engine_name):  # NOQA
```

Comment:
Miss the test skeleton.

Reply:
And does this case test the mounted volume only?

Reply:
What do you mean? the test mounts the volume into the host namespace, so that it tests the fsfreeze implementation.
The sync fallback is tested by all the other snapshot tests.

Reply:
OK, got it. This function won't test the sync fallback.